packages/core/components/Puck/components/Sidebar/styles.module.css

```diff
@@ -0,0 +1,33 @@
+.Sidebar {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  overflow-y: auto;
+}
+
+.Sidebar--left {
+  background: var(--puck-color-grey-12);
+  border-inline-end: 1px solid var(--puck-color-grey-09);
+  grid-area: left;
+}
+
+.Sidebar--right {
+  background: var(--puck-color-white);
+  border-inline-start: 1px solid var(--puck-color-grey-09);
+  grid-area: right;
+}
+
+.Sidebar-resizeHandle {
+  position: absolute;
+  height: 100%;
+}
+
+.Sidebar--left + .Sidebar-resizeHandle {
+  grid-area: left;
+  justify-self: end;
+}
+
+.Sidebar--right + .Sidebar-resizeHandle {
+  grid-area: right;
+  justify-self: start;
+}
```

Comment:
This is not technically SUITCSS format, but it works and I'm happy to refactor it in post, so don't worry about it.

If you're interested, to follow SUITCSS your HTML should have a wrapper like so

```html
<div class="Sidebar Sidebar--left">
  <div class="Sidebar-contents />
  <div class="Sidebar-resizeHandle" />
</div>
```

And your CSS selector would use a modifier descendant, rather than sibling selector

```css
.Sidebar--left Sidebar-resizeHandle {}
```

Reply:
Aah, thanks. I see my issue now.

Reply:
will leave it for now